spec/integration_spec.rb

```diff
@@ -1248,6 +1248,12 @@ class SerializedDocument < ActiveRecord::Base
 end
 
 describe 'People' do
+  it 'adds custom complex attribute' do
+    People.create(first_name: 'Jane', last_name: 'Doe', card_number: 75_801_887)
+    result = People.raw_search('Jane')
+    expect(result['hits'][0]['full_name']).to eq('Jane Doe')
+  end
+
```

Comment:
I moved it because the index has to create first, otherwise the 2 following tests will fail

Reply:
Since tests should be independent of each other, I could guess that our suite is not randomized, let's do that in the future :)

Reply:
I completely agree! the tests should be independent :)